tavern/tomes/netstat/main.eldritch

```diff
@@ -47,7 +47,7 @@ def netstat():
         # Get proc info. "pids" will only ever be one
         if "pid" in n:
             if n["pid"]:
-                fields += ["["+str(n["pid"])+"]"+pids.get(n["pid"])]
+                fields += ["["+str(n["pid"])+"]"+pids.get(n["pid"], "")]
```

Comment:
What's going on here?

Reply:
This just fixes a pylance error. `Operator "+" not supported for types "str" and "Unknown | None"`. By setting the default return of n["pid"] to a string, it will never be none. Its the only change I have to make to remove all pylance errors from the existing tomes.